docs/organization/dynamic-sampling/index.mdx

```diff
@@ -1,18 +1,54 @@
+
+When you use Dynamic Sampling, projects in your organization are sampled based on their recent event volumes. High-volume projects are sampled at lower rates while lower-volume projects are sampled at higher rates or not sampled at all, based on your recent event data. This makes it possible for each project to retain a meaningful set of key events, improving visibility into all projects regardless of their size.
+
+![The dynamic sampling mode and target sample rate setting.](./img/mode-and-target-sample-rate.png)
+
+## Modes
+
+You can choose between two Dynamic Sampling modes for your organization: manual or automatic.
+
+    * **`automatic`** (default): In automatic mode, the sample rate for each project is set automatically based on relative project volume, so events fit within your organization’s billing quota. This boosts visibility into lower-volume projects. To account for any changes in event volume, sample rates are recalibrated on a regular basis. Automatic mode overrides any manually-set project sample rates. 
+    
+    * **`manual`**: If you want to set the exact sample rate for each project in your organization yourself, choose manual mode. Note, that in this mode, there is no automatic recalibration.
+
+
+[]: # (TODO: Rename AM3 with the actual user-facing name.)
+
 <Note>
 
-Retention priorities are only available on the latest version of our plans, starting from 1M reserved transactions. Anyone in your org with owner or billing-level permissions can update your plan by going to Settings > Subscription, and clicking the "Manage Subscription" button.
+Please note that these Dynamic Sampling modes are only available for customers on the AM3 platform.
 
 </Note>
 
-While storing all your data makes sense at relatively low volumes, as your application scales, storing a raw copy of all your data has diminishing returns. When data reaches high volumes, Sentry begins to automatically prioritize retaining certain transactions over others with server-side data retention strategies called retention priorities.
 
-It's important to note that even when we begin to only store the most valuable data based on retention priorities, the [performance metrics](/product/performance/metrics/) you see are still based on **all** the events you send to Sentry. This ensures that you get a full and complete view of your application's health.
+## Differences to Sampling in the Sentry SDKs
+
+Dynamic Sampling is different from the sampling that you can configure in the Sentry SDKs. The SDK sampling rate is applied before events are sent to Sentry, while Dynamic Sampling is applied after events are received by Sentry. This means that Dynamic Sampling can be used to adjust the sample rate for events that have already been sent to Sentry and only take into account the volume of events that have been received by Sentry.
+
+Also in contrast to the Dynamic Sampling, the SDK sample rate is never dynamically adjusted.
```

Comment:
Strictly speaking, "never" isn't true :) `traces_sampler` allows customers to write their own code for dynamically adjusting sample rates. I think the true differences can rather be summarized as:
- Sampling in SDKs affects how much is captured and sent to Sentry. For high throughput scenarios, this may be necessary to reduce instrumentation overheads.
- To change sample rates in SDKs, one has to update the `traces_sample_rate` or `traces_sampler` callback, which usually requires a new deployment or release.
- With dynamic sampling, one can change sample rates without touching the application or SDK. This affects the amount of data _stored_ at Sentry, however, it will not have an effect on how much is sent from the SDKs.
- Dynamic sampling continuously monitors incoming traffic patterns and adjusts fine-grained sample rates within the projects to prioritize more important data. See the sampling priorities for examples.